vpr/src/route/check_rr_graph.cpp

```diff
@@ -173,7 +173,9 @@ void check_rr_graph(const t_graph_type graph_type,
                 bool is_wire = (device_ctx.rr_nodes[inode].type() == CHANX
                                 || device_ctx.rr_nodes[inode].type() == CHANY);
 
-                if (!is_chain && !is_fringe && !is_wire) {
+                if (device_ctx.rr_nodes[inode].capacity() == 0) {
```

Comment:
These changes are not related to your PR, please remove them

Reply:
I don't believe they are exactly related but when we switch to the check_rr_graph.cpp on master vpr crashes on checking the rr_graph we produced.

Reply:
See https://github.com/SymbiFlow/symbiflow-arch-defs/pull/830

Reply:
So is this check expected to fail now?  Or did something change such that the rest of our code no longer works? (I reverted back to the check_rr_graph.cpp on master for now)

Reply:
Ya, in SymbiFlow/symbiflow-arch-defs#830 we removed the need for these capacity = 0 nodes.

Reply:
Thanks, updating symbiflow-arch-defs fixed this issue